tests/matrix_free/copy_feevaluation.cc

```diff
@@ -172,8 +172,7 @@ test()
   stokes_sub_blocks[dim] = 1;
   DoFRenumbering::component_wise(dof_handler, stokes_sub_blocks);
 
-  std::set<types::boundary_id> no_normal_flux_boundaries;
-  no_normal_flux_boundaries.insert(1);
+  std::set<types::boundary_id> no_normal_flux_boundaries = {1};
```

Comment:
```suggestion
  const std::set<types::boundary_id> no_normal_flux_boundaries = {1};
```
I think `const` should work here, too.

Reply:
I agree. I widened my search and found some more appearances.